pkg/installmanager/installmanager.go

```diff
@@ -1188,23 +1192,27 @@ func (m *InstallManager) setupSSHKnownHost(knownHost string) error {
 		return err
 	}
 	defer f.Close()
-	passwdLine := fmt.Sprintf("default:x:%s:0:default user:/home/hive:/sbin/nologin\n", uid)
+	passwdLine := fmt.Sprintf("default:x:%s:0:default user:%s:/sbin/nologin\n", getHomeDir(), uid)
```

Comment:
Did the order here get mixed up? The first `%s` should be the uid and the second should be the home directory, right?

Reply:
Wow, that would now have been fun to debug. Nice catch.

Reply:
Fixed.